2022-03-06 11:58:14,306 INFO    MainThread:695696 [wandb_setup.py:_flush():71] setting env: {}
2022-03-06 11:58:14,307 INFO    MainThread:695696 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-06 11:58:14,307 INFO    MainThread:695696 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-06 11:58:14,307 INFO    MainThread:695696 [wandb_init.py:_log_setup():371] Logging user logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220306_115814-21to2sm9/logs/debug.log
2022-03-06 11:58:14,307 INFO    MainThread:695696 [wandb_init.py:_log_setup():372] Logging internal logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220306_115814-21to2sm9/logs/debug-internal.log
2022-03-06 11:58:14,307 INFO    MainThread:695696 [wandb_init.py:init():404] calling init triggers
2022-03-06 11:58:14,307 INFO    MainThread:695696 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-03-06 11:58:14,307 INFO    MainThread:695696 [wandb_init.py:init():460] starting backend
2022-03-06 11:58:14,307 INFO    MainThread:695696 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-06 11:58:14,314 INFO    MainThread:695696 [backend.py:ensure_launched():216] starting backend process...
2022-03-06 11:58:14,317 INFO    MainThread:695696 [backend.py:ensure_launched():222] started backend process with pid: 695705
2022-03-06 11:58:14,318 INFO    MainThread:695696 [wandb_init.py:init():469] backend started and connected
2022-03-06 11:58:14,320 INFO    MainThread:695696 [wandb_init.py:init():533] updated telemetry
2022-03-06 11:58:14,332 INFO    MainThread:695696 [wandb_init.py:init():563] communicating current version
2022-03-06 11:58:15,374 INFO    MainThread:695696 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-06 11:58:15,374 INFO    MainThread:695696 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-03-06 11:58:15,684 INFO    MainThread:695696 [wandb_init.py:init():606] starting run threads in backend
2022-03-06 11:58:18,277 INFO    MainThread:695696 [wandb_run.py:_console_start():1810] atexit reg
2022-03-06 11:58:18,278 INFO    MainThread:695696 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-03-06 11:58:18,278 INFO    MainThread:695696 [wandb_run.py:_redirect():1689] Redirecting console.
2022-03-06 11:58:18,279 INFO    MainThread:695696 [wandb_run.py:_redirect():1745] Redirects installed.
2022-03-06 11:58:18,279 INFO    MainThread:695696 [wandb_init.py:init():633] run started, returning control to user process
2022-03-06 11:58:20,377 INFO    MainThread:695696 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-03-06 11:58:20,378 INFO    MainThread:695696 [wandb_run.py:_restore():1752] restore
2022-03-06 11:58:23,290 INFO    MainThread:695696 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-03-06 11:58:23,393 INFO    MainThread:695696 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-03-06 11:58:23,670 INFO    MainThread:695696 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-03-06 11:58:23,773 INFO    MainThread:695696 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 15558
}

2022-03-06 11:58:23,875 INFO    MainThread:695696 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 15558
}

2022-03-06 11:58:23,977 INFO    MainThread:695696 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 15558
}

2022-03-06 11:58:24,079 INFO    MainThread:695696 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 15558
}

2022-03-06 11:58:24,180 INFO    MainThread:695696 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15558
  total_bytes: 15558
}

2022-03-06 11:58:24,283 INFO    MainThread:695696 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15558
  total_bytes: 15558
}

2022-03-06 11:58:24,385 INFO    MainThread:695696 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15558
  total_bytes: 15558
}

2022-03-06 11:58:24,488 INFO    MainThread:695696 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15558
  total_bytes: 15558
}

2022-03-06 11:58:24,590 INFO    MainThread:695696 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15558
  total_bytes: 15558
}

2022-03-06 11:58:24,692 INFO    MainThread:695696 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15558
  total_bytes: 15558
}

2022-03-06 11:58:25,015 INFO    MainThread:695696 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15558
  total_bytes: 15558
}

2022-03-06 11:58:25,660 INFO    MainThread:695696 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15558
  total_bytes: 15558
}
local_info {
}

2022-03-06 11:58:26,727 INFO    MainThread:695696 [wandb_run.py:_append_files():2180] logging synced files
